tests/ciscotest.rb

```diff
@@ -176,6 +177,16 @@ def skip_nexus_i2_image?
       Utils.nexus_i2_image
   end
 
+  def system_image
+    @image ||= Platform.system_image
+  end
+
+  def defect?(pattern, msg)
+    return false unless system_image.match(Regexp.new(pattern))
+    puts "\n#{self.class}##{caller[0][/`.*'/][1..-2]} -> NOOP <- [#{msg}]"
```

Comment:
Here's another idea. I removed the `W, [2016-09-06T08:23:07.298520 #7877] ` from the log output and changed `NOOP` to `NOT TESTED`. Is this better?

```
WARN -- : TestVrf#test_description -> NOT TESTED <- [CSCuy36637: Cannot remove description]
TestVrf#test_description = 5.53 s = .
```